insights/specs/jdr_archive.py

```diff
@@ -0,0 +1,45 @@
+from functools import partial
+from insights.specs import Specs
+from insights.core.plugins import datasource
+from insights.core.context import JDRContext
+from insights.core.spec_factory import simple_file, foreach_collect, first_file, glob_file, listdir
+
+first_file = partial(first_file, context=JDRContext)
+glob_file = partial(glob_file, context=JDRContext)
+simple_file = partial(simple_file, context=JDRContext)
+foreach_collect = partial(foreach_collect, context=JDRContext)
+
+
+class JDRSpecs(Specs):
+    # Here use glob_file is to return list type result to keep same with  jboss specs default.py
+    # jdr just collect files under one jboss home,
+    # For insights-client, we will collect files under different jboss home, return list is necessary.
+    jboss_standalone_server_log = glob_file("JBOSS_HOME/standalone/log/server.log")
+
+    @datasource(jboss_standalone_server_log, context=JDRContext, multi_output=True)
+    def jboss_standlone_conf_file(broker):
+        # read the server.log to detect if specific xml is using in the startup command
```

Comment:
Please add a docstring to this datasource to explain what is being collected by this datasource.

Reply:
Yes. will add that.